README.md

```diff
@@ -51,6 +51,12 @@ Now move to App.tsx. Import the page from the component you had just created. an
 <Route path="[PATH THAT SHOWS IN THE URL]" element ={<[YOUR PAGE] />} />
 ```
 
+Due to frontend dependencies with viewing markdown, make sure you run:
+
+```bash
+npm install react-markdown remark-gfm github-markdown-css prismjs

```

Comment:
Shouldnt this be included in the initial npm install during environment setup? This is more of an update/dev requirement than something for the docs

Reply:
Fixed this in my branch.